### PR TITLE
Append 'slot' to extensionSlots to comply with naming convention

### PR DESCRIPTION
**Pull request by donaldkibet**
### What does this PR do?

1. Update slot names to comply with the naming convention on slots

### CODE REVIEW

**Review by FlorianRappl — COMMENTED** [review]
LGTM - thanks!

File: src/index.ts [hunk 1]
```diff
@@ -61,15 +61,15 @@ function setupOpenMRS() {
     extensions: [
       {
         id: "location-picker",
-        slot: "location-picker",
+        slot: "location-picker-slot",
```
[the remark this answers is not in the document]
Comment (donaldkibet) [reply src/index.ts]: reverted it to the original name, not sure on the most appropriate name

**Comment by donaldkibet** [comment]
> Hm what about the issue with the `slot: "location-picker",` @donaldkibet ? Would be nice to have a better name here, too (and one that complies with the `-slot`). Thanks for your contribution on this one - much appreciated 🍻!

no issue with the name, I couldn't suggest a better name.

**Comment by FlorianRappl** [comment]
Can you change it @donaldkibet ?

**Pull request merged**